tapir/coop/templates/coop/log/create_payment_log_entry.html

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+{% blocktranslate with amount=entry.amount payment_date=entry.payment_date paying_member=entry.paying_member.get_info.get_display_name %}New Payment: {{amount}} € on {{payment_date}}{% endblocktranslate %}
```

Comment:
There's still a reference to the paying member here.

Reply:
yes, seen it too after commit